dspace/config/spring/api/core-services.xml

```diff
@@ -170,13 +170,14 @@
         <property name="separator">
             <value>, </value>
         </property>
+        <property name="useForPlace" value="true"/>
```

Comment:
Is there any reason why a virtual metadata should not be *used for place*? Or is this just for performance if there are a lot of relations?

Reply:
This is both for performance and because a single relationship can map multiple metadata fields at once while calculation can only happen on one field (it is a place stored in the relationship)

If you have a relationship between an author and a publication, there's no need to start calculating the publication's place in the author using a metadata field in the author